meta-phosphor/recipes-phosphor/settings/phosphor-settings-defaults/defaults.yaml

```diff
@@ -119,6 +119,12 @@
           TPMEnable:
              Default: 'false'
 
+/xyz/openbmc_project/control/host0/TPMDisable:
+    - Interface: xyz.openbmc_project.Control.TPM.Policy
+      Properties:
+          TPMDisable:
+             Default: 'true'
```

Comment:
We want the default to be 'false', meaning that the TPM is not disabled by default, only if the customer disables it.